src/main/java/Fenix.java

```diff
@@ -0,0 +1,124 @@
+import java.util.Scanner;
+
+public class Fenix {
+
+    public static final int HORIZONTAL_LINE_USER_COMMAND_LENGTH = 60;
+    public static final int HORIZONTAL_LINE_FENIX_MODIFICATION_LENGTH = 52;
```

Comment:
Good use of constants to prevent the use of magic numbers in the code base. Excellent job!